staging/src/k8s.io/apiserver/pkg/server/filters/maxinflight_test.go

```diff
@@ -101,6 +102,8 @@ func TestMaxInFlightNonMutating(t *testing.T) {
 	waitForCalls := true
 	waitForCallsMutex := sync.Mutex{}
 
+	fcmetrics.Register()
```

Comment:
Can you instead Register those in createMaxInflightServer?

Reply:
Done in #110516